MoreLinq/MoreLinq.csproj

```diff
@@ -241,6 +242,7 @@
   <ItemGroup>
     <AdditionalFiles Include="PublicAPI/$(TargetFramework)/PublicAPI.Shipped.txt" />
     <AdditionalFiles Include="PublicAPI/$(TargetFramework)/PublicAPI.Unshipped.txt" />
+    <Content Include="Pairs.cs" />
```

Comment:
This is not necessary - `.cs` files are included automatically. Please review your Visual Studio settings to understand why it added this line; and remove this line.

Reply:
True. Thanks.